app/assets/javascripts/eventscripts/create_event.js

```diff
@@ -36,6 +87,19 @@ $( document ).on('turbolinks:load', function() {
         }
     }
 
+    function addDays(date, days) {
+        var result = new Date(date);
+        result.setDate(result.getDate() + parseInt(days));
+        var dd = result.getDate();
+        var mm = result.getMonth()+1;
+        var y = result.getFullYear();
+        if(mm < 10){
+            mm = "0"+ mm;
+        }
+        var formattedDate = y + '-' + mm + '-' + dd;
```

Comment:
This `formattedDate` has a different Syntax than the other `formattedDate` further down (ymmdd vs ddmmy) which should be consistent (ddmmy)